models/models.technicians.js

```diff
@@ -17,3 +17,13 @@ exports.findTechnician = async (id) => {
     console.error(e);
   }
 };
+
+exports.updateTechnician = async (id, updates) => {
+  try {
+    await User.updateOne({_id: id}, { $set: {"technician.services": updates.services} })
+    const updatedTechnician = await User.findById(id);
```

Comment:
Double check `updateOne` doesn't return the updated object, if it does return it we can skip this `findById` call and assign `updatedTechnician` as the return value of `updateOne`. Otherwise this is fine